index.html

```diff
@@ -4,7 +4,64 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap"
+      rel="stylesheet"
+    />
     <title>setup-mb-skeleton</title>
+    <link rel="stylesheet" href="style.css" />
   </head>
-  <body></body>
+  <body>
+    <header>
+      <h1>My logo</h1>
```

Comment:
- [ ] `My logo` is supposed to be a logo, and usually, logo is a link that takes users to the homepage. Please wrap Arturo Hermida with  an `<a> `tag
- [ ] Please try to match your design with the one in Figma, currently logo and humburger icon on your design are aligned on two sides of the screen :
 
Your design  | Figma design
------------- | -------------
<img width="795" alt="Screenshot 2022-03-17 at 15 55 34" src="https://user-images.githubusercontent.com/77038610/158803443-d4dfad88-5f6a-4224-b260-44ac64595c14.png"> | <img width="325" alt="Screenshot 2022-03-17 at 15 56 12" src="https://user-images.githubusercontent.com/77038610/158803562-8a4a76c7-1b2a-44d6-86f1-3915b4ef0267.png">

Reply:
Maybe you should consider only the mobile version, as I can see here on this picture you consider a desktop size of my design, you know we didn't have the requirements to use media.

Reply:
- [ ] Hi @christianbiring1, in my video below, I demonstrated that I was not able to see anything when I set the mobile width. Please check the video one more time, On mobile width the whole page was getting white 😢